### DISCUSSION TITLE
v5 styles

### DISCUSSION THREAD

**Discussion by NewbLady** [willwulfken/MidJourney-Styles-and-Keywords-Reference#40] [Ideas and Style Submissions]
hello

first of all, thank you for this reference resource I've been using it since I started MJ. excellent work. 

secondly, I've never used GitHub before. it's a bit confusing if you don't use such things. I don't know how to do pull requests, branches or forks. none of that. I can learn some although I don't know where to start.
I want to help generate images in V5. I've done some today. pulled styles directly off of v4 keywords. seed 4776 for all. 
also would like to add some overtime that isn't there like glacier striations. I pulled out a mining glossary in v4. it was handy.

i put the images in this drive. if you could help me a bit more than others so I can put them where they go.

 https://drive.google.com/drive/folders/1CvoqbdUs-oiSEmdgfPoZ6yGxQ8Xs6732?usp=share_link

Thank you

**Comment by willwulfken** [comment]
Thank you so much! I'm very curious to look at all of these... This is prefect because I just figured out the perfect way to make the image size smaller without quality loss.

**Reply by NewbLady** [reply]
Ok. I have more to add to the drive. Plus I'm doing more today. 
I've got materials, dimensions, and lighting done.
Do you want me to just add to Google drive, let you know and you can do something with it. Unless you want me to figure out GitHub.

**Comment by NewbLady** [comment]
I've finished materials, dimensionality, lighting, drawing and art mediums, perspective, themes and design styles. though not all are in the drive. I'm getting a new computer this weekend so ill more stuff around and add it next week.

**Reply by willwulfken** [reply]
That's amazing! I know I generate a lot of images but I think you've beat me at the pace of which you've generated so many. How'd you do it?

**Reply by NewbLady** [reply]
just did it on alpha site in fastmode

**Reply by NewbLady** [reply]
also did some artists tonight. i don't know if you want those or not. i got the names off of the Great Big List Of Midjourney Artist Styles google sheet. you could look see https://www.midjourney.com/app/users/69dd4485-a229-4879-ad77-630aa8cd43b7/ let me know what more you would like.

**Reply by willwulfken** [reply]
Anything you'd like to submit I'd be happy to look at. I've already added between 1300 and1500 of them. Credited you on each page as well.

**Comment by NewbLady** [comment]
i got done digital, camera, film, lenses, geometry, geography and culture today. i added a few to geography and culture. don't know if you want to add them. also played around with a few of these. wow there are lots. i have no desire to do them all but it fun. and now I know they are out there. there's a few tiling ones that I did with --tile and without.  https://en.wikipedia.org/wiki/List_of_polygons,_polyhedra_and_polytopes
can I remove what's in the google drive and add these? i have my cloud drives mostly full so I can change over computers this week. unless you want to pull them right off of my midjourney user page. just let me know.

**Reply by willwulfken** [reply]
I downloaded them once more to be sure I didn't miss any so I'm all set with those. You can delete them and upload the new ones if you'd like. Thank you for all of this and making sure I've gotten them all.

**Comment by NewbLady** [comment]
bunch more are in the drive. i was able to clear a bit more space in the drive. however, ill be running more prompts today. whenever you have time. let me know when you've got all those.

**Comment by NewbLady** [comment]
I've got all that I got done added to the drive. I've got most of the categories done. Still have objects, animals and nature. All three will be long. I'll work at it though. However, wonn't be finishing up for a bit. Computer parts all came in. Now. New computer!!!

**Reply by willwulfken** [reply]
I downloaded all of the new ones.
I hope you enjoy that new computer! I'd love to see it if you want to send a picture. Very interested because I built my own a while ago.
Take all the time you need I really can't overstate how much I appreciate this.

**Reply by willwulfken** [reply]
~~Quick question. Were the ones I just downloaded were generated in the past few days rather than today? Also, will the ones you upload next be from yesterday or today?~~

~~Only asking because I noticed an update in V5 and want to make sure they go in the correct places.~~

The change that I noticed was just a strange anomaly. This has been happening every once in a while.

**Reply by NewbLady** [reply]
just in case, i did most of those today. some are from previously that i wasn't able to add to google drive. the majority is from today.

**Reply by willwulfken** [reply]
Thank you

**Reply by willwulfken** [reply]
I also sent you a gmail about something related but separate. I just don't want to send it here, but you'll see why.

**Comment by NewbLady** [comment]
i have more loaded in the drive. there are additional folders that you can choose to look at or not. furniture types, furniture designers, styles of architecture and such. iève yet to go through those and take out ones that aren't recongnized by MJ. some of the older buildings kinds and ethenic structures aren't there. i'll have to play around more with those prompts.

**Reply by NewbLady** [reply]
let me know if anything is missed. idk if it's all done. it might be though

**Reply by willwulfken** [reply]
I've been generating the misspellings and some of the missing ones. There haven't been too many.

In fact, some of the misspelled ones are very interesting to me and I want to find a way to add them. I'm also still working towards adding the ones I don't have but I want to generate them in V4 too. I've just been a bit busy lately is all. The ones I have are easy to add because of the regexes I've created to help with that.

Thank you very much! I'll let you know when I download them.

**Reply by willwulfken** [reply]
Alright it seems that they have all downloaded correctly so I've got them

**Reply by NewbLady** [reply]
great. 
i could have copied and pasted for less typos but i was trying to practice typing. i had to wait for the rendering so it didn't take that much more time. now im used to my new keyboard at least.

**Reply by willwulfken** [reply]
It's all good. It's not that many. Plus, it's interesting to see what they generate.

**Reply by willwulfken** [reply]
On top of that, this has helped me find some of my typos. I'm not sure if you realized it or not, but some of your images were correct even though a few of mine were typos.

**Comment by CameronKnowlton** [comment]
Ooh, Furniture, I'd be interested in those. Thanks NewbLady!

Cameron Knowlton
iGods Internet Marketing Inc.
***@***.***
P: 250.382.0226

**Reply by NewbLady** [reply]
furniture is from these lists 
https://en.wikipedia.org/wiki/List_of_furniture_types
https://en.wikipedia.org/wiki/List_of_furniture_designers

**Comment by NewbLady** [comment]
So... Is everything that's prompting in for v5 now obsolete? I haven't taken the time to check everything.

**Reply by willwulfken** [reply]
No, most of the changes are minor. I'd keep those pages up either way. Much like V4, the images are still super helpful and in actuality, even though I use a consistent seed, a lot of people do not. That would be similar to showing a different prompt seed here even if V5 was updated more.
